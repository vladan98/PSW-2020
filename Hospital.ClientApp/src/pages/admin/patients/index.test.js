import Patients from "."
import render from "../../../helpers/renderComponent"

describe("Patients", () => {
    it("should render patients page", () => {
        const { container } = render(<Patients />)

        expect(container).toMatchSnapshot()
    })
})