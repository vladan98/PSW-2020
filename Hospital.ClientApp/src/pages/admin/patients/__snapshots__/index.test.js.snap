// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Patients should render patients page 1`] = `
<div>
  <main
    class="MuiContainer-root MuiContainer-maxWidthLg"
  >
    <div
      class="makeStyles-paper-1"
    >
      <div
        align="center"
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          class="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
          role="progressbar"
          style="width: 40px; height: 40px;"
        >
          <svg
            class="MuiCircularProgress-svg"
            viewBox="22 22 44 44"
          >
            <circle
              class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
              cx="44"
              cy="44"
              fill="none"
              r="20.2"
              stroke-width="3.6"
            />
          </svg>
        </div>
      </div>
    </div>
  </main>
</div>
`;
